src/status_im/ui/screens/signing/views.cljs

```diff
@@ -240,13 +245,19 @@
                :title :t/send-request-amount
                :error amount-error
                :accessories [[acc-text (if amount (str amount) "0")
-                              (or display-symbol fee-display-symbol)]]}]
+                              (or display-symbol fee-display-symbol)] "·" (if converted-value [react/nested-text {:styles {:color :colors/black}} (str "~" (i18n/format-currency converted-value (:code wallet-currency)))]
+                                                                              [react/activity-indicator {:color   :colors/gray
+                                                                                                         :ios   {:size  :small}
+                                                                                                         :android {:size :16}}]) (:code wallet-currency)]}]
              [separator]])
           [list-item/list-item
            {:type        :small
             :title       :t/network-fee
             :error       gas-error
-            :accessories [[acc-text fee fee-display-symbol] :chevron]
+            :accessories [[acc-text fee fee-display-symbol] "·" (if converted-fee-value [react/nested-text {:styles {:color :colors/black}} (str "~" (i18n/format-currency converted-fee-value (:code wallet-currency)))]
```

Comment:
don't use `acc-text` here, if you look at the definition above in this namespace it is just a nested-text with 2 parts and here you have more. You also misused nested-text here, you should use nested-text instead of acc-text. here is an example of how to use it:

https://github.com/status-im/status-react/blob/496f3f1cc78703c3d629f9f458a0e966b04e5c61/src/status_im/ui/screens/privacy_policy/views.cljs

as you can see you give a global style and then you can have as many arguments as you want, strings or vector, the strings get the global style and the vectors are tuples of style + string, the style is merged with the global style so you only have to put what changes there